src/db/columnSets.ts

```diff
@@ -55,7 +55,7 @@ export const SamplesColumnSet = new pg.helpers.ColumnSet([
 ], { table: new pg.helpers.TableName({ table: "samples", schema: "jtl" }) })
 
 export const MonitorColumnSet = new pg.helpers.ColumnSet([
-  "timestamp", "name", "cpu", {
+  "timestamp", "name", "cpu", "mem", {
```

Comment:
This is a breaking change, if "mem" wont be provided, the API will throw 500

Reply:
I've created new MR with v4: https://github.com/ludeknovy/jtl-reporter-listener-service/pull/50